test/versioned/aws-sdk-v3/redshift-data.test.js

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+const assert = require('node:assert')
+const test = require('node:test')
+const { createEmptyResponseServer, FAKE_CREDENTIALS } = require('../../lib/aws-server-stubs')
+const common = require('./common')
+const helper = require('../../lib/agent_helper')
+const { match } = require('../../lib/custom-assertions')
+
+test('Redshift-data', async (t) => {
```

Comment:
What's being used to make this relationship? I don't see it being asserted

Reply:
Updated tests to assert that `host` attribute exists on segments which is setting `server.address` that "should" be used to make the relationship.